### PR TITLE
Bug 1179343 - Wi-Fi hotspot layout to show enabled/disabled messages.…

### PR DESCRIPTION
**Pull request by scottwu**
… r=gasolin

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2fa8f98ebcccc546c9472dd8fd87cac29089df8f

File: apps/settings/elements/hotspot.html [hunk 1]
```diff
@@ -13,25 +13,13 @@ <h2 data-l10n-id="internetSharing-wifi"></h2>
         <li>
           <gaia-switch id="tethering-wifi-enabled">
             <label data-l10n-id="wifi-hotspot"></label>
+            <details hidden id="wifi-hotspot-enabled-msg"></details>
+            <details hidden id="wifi-hotspot-disabled-msg" data-l10n-id="wifi-hotspot-disabled-msg"></details>
```
Comment (gasolin) [inline apps/settings/elements/hotspot.html]: please use single `wifi-hotspot-msg` and use `setAttribute('data-l10n-id', string)` for enabled / `removeAttribute('data-l10n-id')` for disabled cases

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -531,7 +531,8 @@ internetSharing-header={{internetSharing}}
 internetSharing-usb=USB
 internetSharing-wifi=Wi-Fi
 wifi-hotspot=Wi-Fi Hotspot
-wifi-name=Name
+wifi-hotspot-enabled-msg=Hotspot "{{hotspotSSID}}" is enabled
+wifi-hotspot-disabled-msg=Disabled
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: we can reuse common `Disabled` attribute here

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=280aba0f9fda3aa3228ce5027bf7d4d2e3ec5554

**Pull request merged**